README.md

```diff
@@ -1,12 +1,11 @@
-RIOT Release Specs
-==================
+# RIOT Release Specs
+
```

Comment:
Why?

Reply:
Consistency, I could not judge the heading size with the mix.

Reply:
1. This is normal Markdown style and used in other READMEs as well (and better readable as headings in my opinion)
2. This is a separate change and should go into its separate PR, but in the end it is a question of taste.

Reply:
ok, removed